libfreerdp/core/pkcs11/cert_info.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * PKCS #11 PAM Login Module
```

Comment:
@dodo040 Well, it still is quite a dependency and if not maintained upstream it will probably not be maintained here :/